source/publishing_data/04_configuring_a_source/connectors/openagenda.rst

```diff
@@ -0,0 +1,58 @@
+OpenAgenda connector
+====================
+
+.. admonition:: Important
+   :class: important
+
+   This connector is not available by default. Please contact Opendatasoft for the activation of this connector on a given Opendatasoft domain.
+
+The OpenAgenda connector extracts events from an agenda, using the version 2 of the OpenAgenda REST API.
```

Comment:
```suggestion
The OpenAgenda connector extracts events from an agenda, using the OpenAgenda REST API.
```

No need to specify that we are using the version 2 of their API (as it's not the case and as this is an internal implementation detail).

Reply:
Would be interesting to add a link to the OpenAgenda web site (or documentation)?